examples/platform/k32w/app/support/FreeRtosMbedtlsUtils.c

```diff
@@ -78,3 +78,13 @@ void * pvPortCallocRtos(size_t num, size_t size)
 
     return p;
 }
+
+void * otPlatCAlloc(size_t aNum, size_t aSize)
+{
+    return pvPortCallocRtos(aNum, aSize);
+}
+
+void otPlatFree(void * aPtr)
+{
+    return vPortFree(aPtr);
+}
```

Comment:
This is for platform support of OpenThread. Why is it in *MbedtlsUtils.c ?

Reply:
Agree, the naming is not correct. I moved otPlatCAlloc and otPlatFree to src/platform/K32W/ThreadStackManagerImpl.cpp.